forms/admin.py

```diff
@@ -0,0 +1,71 @@
+from django.contrib.admin import ModelAdmin
+from django.contrib.contenttypes.models import ContentType
+from django import forms
+from django.forms import ModelForm
+
+from formula_one.models import (
+    ContactInformation,
+    LocationInformation,
+    SocialInformation,
+)
+
+admin_form_dict = {
+    'ContactInformation': ContactInformation,
+    'LocationInformation': LocationInformation,
+    'SocialInformation': SocialInformation,
+}
```

Comment:
@dhruvkb This dictionary will eventually hold all the admin forms but it is initalized with the models because line 56 (`model = admin_form_dict[class_name]`) required the model class not the string.
This method seems hacky to me. Please tell me if there is a better method to do the same.